core/capabilities/webapi/outgoing_connector_handler_test.go

```diff
@@ -328,6 +329,142 @@ func TestHandleSingleNodeRequest(t *testing.T) {
 	})
 }
 
+func Test_handleSingleNodeRequest(t *testing.T) {
```

Comment:
note that `Test_handleSingleNodeRequest` doesn't test any of the logic of sending an actual request.  Those tests are in `TestHandleSingleNodeRequest` which uses a real `sendRequest` function.